### DISCUSSION TITLE
Personal Data

### DISCUSSION THREAD

**Discussion by taaib** [m-ld/m-ld-spec#50] [Discussion]
What (if any) restrictions can be placed on data to ensure that only a particular data subset can be shared across the clones i.e. some “personal” stays on one clone only?

**Comment by gsvarovsky** [comment]
Let me start with this: https://github.com/m-ld/m-ld-spec/discussions/48#discussioncomment-192966
> For now, let's say that **m-ld** is _not_ a database, and definitely not in the sense of a 'server-based repository of truth'. Instead it's a portable thing that lives _where the data is being used_. It's like a combination of a local file or document, and the in-memory structured representation of that file's contents; like the 'Model' in MV[C|P]. Except, of course, it's shared, live, with others also operating on the same data

So, in general **m-ld** can be a part of an app's architecture that handles some live datasets: the 'domains'. The app can operate using any number of domains, at the same time, and these domains can have different security applied to them by the app. So, you could give each user a 'personal' data domain which is only accessible to them. (Or you could choose to store that data in a database.)

On the other hand we're also working on a way to do fine-grained access control even with in a domain. So watch this space!

Thanks for the question, @taaib
